### PR TITLE
Add doc for harvesting simple URL

### PR DESCRIPTION
**Pull request by tkohr**
PR adds documentation for harvesting via a simple URL harvester, with examples of opendatasoft and ESRI configurations.

### CODE REVIEW

**Review by fgravin — APPROVED** [review]
Very good @tkohr thanks !
Maybe you could just add the dkan settings as you use it as well.

Anything to add @fxprunayre @archaeogeek ?
It would be great to add settings for rdf or xml harvesters as well.
I would be particularly interested in CKAN settings.

Thanks to all

File: source/user-guide/harvesting/harvesting-simpleurl.rst [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+- **Site** - Options about the remote site.
+
+    - *Name* - This is a short description of the remote site. It will be shown in the harvesting main page as the name for this instance of the harvester.
+    - *Service URL* - The URL of the server to be harvested. This can include pagination params like ``?start=0&rows=20``
+    - *loopElement* - Propery/element containing a list of the record entries. (Indicated as an absolute path from the document root.) eg. ``/datasets``
+    - *numberOfRecordPath* : Property indicating the total count of record entries. (Indicated as an absolute path from the document root.) eg. ``/nhits``
+    - *recordIdPath* : Property containing the record id. eg. ``datasetid``
+    - *pageFromParam* : Property indicating the first record item on the current "page" eg. ``start``
+    - *pageSizeParam* : Property indicating the number of records containned in the current "page" eg. ``rows``
+    - *toISOConversion* : Name of the conversion schema to use, which must be available as XSL on the GN instance. eg. ``OPENDATASOFT-to-ISO19115-3-2018``
```
Comment (archaeogeek) [inline source/user-guide/harvesting/harvesting-simpleurl.rst:20]: Should we put something here about how conversion schema can come from different schema, and how they should be named in that case? eg `schema:iso19115-3.2018:convert/fromJsonOpenDataSoft` refers to the `fromJsonOpenDataSoft` xsl in the `convert` folder for the i`so19115-3.2018` schema

Comment (archaeogeek) [reply source/user-guide/harvesting/harvesting-simpleurl.rst:20]: @fgravin aside from my comment above, I think this looks good. I can't add anything about ckan settings though as I'm still struggling to get that one to work!

**Review by archaeogeek — COMMENTED** [review]
I just have the one comment about the `toIsoConversion` description, otherwise it looks good

**Comment by fgravin** [comment]
@tkohr I have changed the target branch to `4.0.x`, so you need to restore your branch over it.
Also please adress Jo comment, thanks !

**Comment by fgravin** [comment]
Thanks you @tkohr

**Pull request merged**